slack-announcement-scheduler/db/migrate/20221002090614_create_messages.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
+      t.text :name
+      t.text :description
+      t.text :department
+      t.text :when_to_announce
```

Comment:
Suggest using datetime to store time format
```suggestion
      t.datetime :when_to_announce
```

Reply:
And agree with WY
This should be named like `announce_time`